projects/sandbox/sampling/run.sh

```diff
@@ -0,0 +1,2 @@
+pinto run sample-with-flow --typeo ../pyproject.toml script=sample-with-flow flow=maf embedding=resnet 
```

Comment:
This is a good idea. Do you want to add your runner script for the train project as well